android/release_checklist.md

```diff
@@ -0,0 +1,53 @@
+# Android Development/QA Release Checklist
+
+## Summary
+
+This is a list of important items to consider for developers and QA before releasing an Android app to a store.
```

Comment:
In practice, how do you foresee this checklist being used? For example, is the release manager or QA expected to step through each item on the list before each release? Should developers go through the checklist when making a patch? Should this be used when setting up a new project to ensure we have a test case (manual or automated) for each item on the list?

Personally, I think a lot of these "checklist" items can be integrated into each app's Testrail test cases (they're manual test cases that QA runs for each release – [example: Fenix](https://testrail.stage.mozaws.net/index.php?/runs/view/18275&group_by=cases:section_id&group_order=asc&group_id=47243) – and they're like a super-checklist). This makes me think this could be a good resource for setting up new Android apps but I'm concerned that devs will not think to look for these docs when creating a new project. Is this what you were thinking? If so, do you know how to address this, "no one saw the documentation" problem?

We should probably check with QA how they set up the Testrail test cases for new apps too and see if there is any overlap here.